docs/agents/ups.rst

```diff
@@ -53,10 +53,13 @@ example docker-compose service configuration is shown here::
       - SITE_HUB=ws://127.0.0.1:8001/ws
       - SITE_HTTP=http://127.0.0.1:8001/call
       - LOGLEVEL=info
+    restart: unless-stopped
```

Comment:
Is this needed? Host Manager should also still have the 'up' state, which should automatically restart the agent as well (and has the added perk that if the agent isn't in a Docker container still works.) Though I guess then we need to also note this only works if managed by the Host Manager.

Reply:
I think this is still needed, the default is ``restart: no`` which the Host Manager would still read in from docker-compose